ts/main.ts

```diff
@@ -183,13 +194,15 @@ async function fetchExerciseSearchData(term: string): Promise<void> {
     }
     if (exerciseObjArr.length > 0) {
       exerciseObjArr.forEach((element) => {
-        $cardList.appendChild(renderExercises(element));
+        $exercisesCardList.appendChild(renderExercises(element));
       });
       $noResults?.classList.add('hidden');
     } else {
       $noResults?.classList.remove('hidden');
     }
-    $exercisesNodeList = document.querySelectorAll('.card');
+    const $exercisesNodeList = document.querySelectorAll(
+      '#exercises-card-list > .card',
+    );
     if (!$exercisesNodeList) throw new Error('no exercise nodelist found');
   } catch (error) {
     console.log(error);
```

Comment:
Should have brought this up earlier but we should use `console.error(error)` for our errors